influxql/engine.go

```diff
@@ -669,54 +669,66 @@ func ReduceMean(key Key, values []interface{}, e *Emitter) {
 
 // MapMin collects the values to pass to the reducer
 func MapMin(itr Iterator, e *Emitter, tmin int64) {
-	var values []float64
+	var min *float64
```

Comment:
If you want to ensure we can do that, then have a bool for `pointsYielded` and set that to true. Then only emit if that is true.